freeCodeCamp/freeCodeCamp
# fix(tools): seed MS account when seeding certified user

Checklist:






This PR:
- Adds a script for seeing MS account, and runs it as part of the `seed:certified-user` command
- Closes #52510

I haven't touched MongoDB for years, so please scrutinize the code.

## Steps to test

- Run `pnpm run seed:certified-user`
- Go to `/learn/foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/trophy-write-your-first-code-using-c-sharp`
- Confirm that the page now displays the user's MS account instead of displaying a form

<img width="1680" alt="Screenshot 2024-01-05 at 14 35 36" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/4f85df12-4a19-4219-8a53-b9111649d154">
